content/ecosystem/_index.html

```diff
@@ -0,0 +1,465 @@
+        </div>
+      </div>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-star"></i> cuSpatial</h2>
+          <div class="tags-container border-top has-border-white">
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> GEOPANDAS </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> GEOSPATIAL </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PYTHON / C++ </span>
+          </div>
+          <p class="has-text-white">cuSpatial is a library for lighting-fast spatial analytics. Fully integrated with GeoPandas and growing fast, this toolkit will feel familiar to Python users of GIS tools.</p>
+          <a href="https://github.com/rapidsai/cuspatial">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/cuspatial/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://github.com/orgs/rapidsai/projects/41/views/5">Roadmap <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+          <br><br>
+
+          <h3 class="has-text-white">libcuspatial</h3>
+          <p class="has-text-white">libcuspatial provides the CUDA C++ building blocks powering cuSpatial.</p>
+          <a href="https://github.com/rapidsai/cuspatial">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/libcuspatial/stable">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-star"></i> RAFT</h2>
+          <div class="tags-container border-top has-border-white">
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PRIMITIVES </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> ALGORITHMS </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> CUDA / PYTHON / C++ </span>
+          </div>
+          <p class="has-text-white"> RAFT contains CUDA accelerated primitives for rapidly composing analytics, and is used by other RAPIDS libraries. Its building blocks include dense algebra, sparse algebra, spatial computations, clustering, solvers, and statistics. </p>
+          <a href="https://github.com/rapidsai/raft">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/raft/stable">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-terminal"></i> cuxfilter </h2>
```

Comment:
We switch the icon from `fa-star` to `fa-terminal` from cuxfilter on down, but there are no longer two sections. Are these meant to be distinguished or are they supposed to be the same? I'd use `fa-terminal` everywhere.